README.md

```diff
@@ -23,9 +23,42 @@ Then, instead of the `run.sh` script, run Maven directly:
 $ mvn clean install -Pnative
 ```
 
+## Using the `newdoc` script to generate new pages
+
+Che documentation follows guidelines of the Modular Documentation initiative: https://redhat-documentation.github.io/modular-docs/
+
+You can use `newdoc` to generate templates for the new documentation.
+To learn more about the script: https://github.com/redhat-documentation/tools/tree/master/newdoc
+
+### Using `newdoc` from the `che-docs` container
+`newdoc` is built into the `che-docs` container.
+
+To generate new documentation module with the script, run from the `che-docs` root git directory:
+
+* Create a new assembly in a directory `directory_name`, with a title `your_title`
```

Comment:
Given first feedback received, `directory_name` need a more precise explanation. It can be one of the guides in the list: 
* administration-guide
* contributor-guide
* end-user-guide
* extensions
* installation-guide
* overview